index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the delay
+              between the moment the implementation sends a buffer to the
+              system and the audible sound from the audio output (speaker or
+              connector).
+            </p>
+            <p>
+              The <a><code>latency</code></a> attribute value depends on the
+              platform and the hardware audio output device, the attribute
+              value does not change for the context's lifetime as long as audio
+              output device remains the same. If the audio output device is
+              changed the <a><code>latency</code></a> attribute value will be
+              updated accordingly.
+            </p>
+          </dd>
+          <dt>
+            readonly attribute AudioTimestamp devicePosition
```

Comment:
I keep reading "devicePosition" as the physical position of the device.  Can we choose a different name that's more indicative of the time nature of the attribute?